v1.1/build-a-ruby-app-with-cockroachdb-activerecord.md

```diff
@@ -29,6 +29,8 @@ To install ActiveRecord as well as the [pg driver](https://rubygems.org/gems/pg)
 $ gem install activerecord pg activerecord-cockroachdb-adapter
 ~~~
 
+{{site.data.alerts.callout_success}}The exact command above will vary depending on the desired version of activerecord. Specifically, version 4.2.x of ActiveRecord requires version 0.1.x of the adapter; version 5.1.x of ActiveRecord requires version 0.2.x of the adapter.{{site.data.alerts.end}}
```

Comment:
nit: `activerecord` > `ActiveRecord` at end of the first sentence.

Reply:
Done